sky/templates/runpod-ray.yml.j2

```
@@ -0,0 +1,110 @@
+cluster_name: {{cluster_name}}
+
+# The maximum number of workers nodes to launch in addition to the head node.
+max_workers: {{num_nodes - 1}}
+upscaling_speed: {{num_nodes - 1}}
+idle_timeout_minutes: 60
+
+provider:
+  type: external
+  module: sky.skylet.providers.runpod.RunPodNodeProvider
+  region: {{region}}
```

We need to quote this region, as there is a region named `NO` in runpod catalog, causing some parsing issue for yaml (parsed to `false`)
```suggestion
  region: "{{region}}"
```